<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendOnlineOnly</class>
 <widget class="QWidget" name="SendOnlineOnly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>748</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>60</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>470</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>470</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="control::MwcLineEditNormal" name="sendEdit">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>270</y>
          <width>417</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Recipient wallet address. Note, your peer wallet needs to be online to accept the payment.</string>
        </property>
        <property name="statusTip">
         <string>Recipient wallet address. Note, your peer wallet needs to be online to accept the payment.</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="placeholderText">
         <string>Send to</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="contactsButton">
        <property name="geometry">
         <rect>
          <x>448</x>
          <y>270</y>
          <width>97</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Pick a contact to get the receiver's wallet address</string>
        </property>
        <property name="statusTip">
         <string>Pick a contact to get the receiver's wallet address</string>
        </property>
        <property name="text">
         <string>Contacts</string>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="contactNameLable">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>310</y>
          <width>539</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>      Contact: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="sendButton">
        <property name="geometry">
         <rect>
          <x>188</x>
          <y>428</y>
          <width>170</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Send MWC</string>
        </property>
        <property name="statusTip">
         <string>Send MWC</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="formatsLable">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>310</y>
          <width>539</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>3000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#formatsLable{color: rgba(255,255,255, 0.7);}</string>
        </property>
        <property name="text">
         <string>FORMATS:    [mwcmqs://]&lt;mqs_address&gt;  [http://]&lt;tor_address&gt;[.onion]    http(s)://&lt;host&gt;:&lt;port&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
       <widget class="control::MwcComboBox" name="accountComboBox">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>2</y>
          <width>407</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Select account to send MWC coins from</string>
        </property>
        <property name="statusTip">
         <string>Select account to send MWC coins from</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="amountEdit">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>66</y>
          <width>275</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Amount of MWC coins to send</string>
        </property>
        <property name="statusTip">
         <string>Amount of MWC coins to send</string>
        </property>
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="placeholderText">
         <string>Amount</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="allAmountButton">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>66</y>
          <width>103</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Send entire MWC balance from this account</string>
        </property>
        <property name="statusTip">
         <string>Send entire MWC balance from this account</string>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
       <widget class="control::MwcCheckBox" name="generatePoof">
        <property name="geometry">
         <rect>
          <x>144</x>
          <y>124</y>
          <width>359</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>The proof contains the full transaction and can be viewed by anyone. Keep it if you might need to resolve a dispute; skip it if privacy matters more</string>
        </property>
        <property name="statusTip">
         <string>The proof contains the full transaction and can be viewed by anyone. Keep it if you might need to resolve a dispute; skip it if privacy matters more</string>
        </property>
        <property name="text">
         <string>Generate Transaction Proof</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="label_4">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>168</y>
          <width>547</width>
          <height>83</height>
         </rect>
        </property>
        <property name="text">
         <string>- Transaction won't be sent if the destination wallet is offline and isn't listening for the chosen address.
- Your funds remain locked until the transaction is fully confirmed on-chain.
- You can cancel any unconfirmed transaction to unlock your funds at 'Transactions'.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>16</y>
          <width>103</width>
          <height>95</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_desktop.qrc">:/img/SendOnline@2x.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="descriptionEdit">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>360</y>
          <width>545</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Optional note the recipient will see</string>
        </property>
        <property name="statusTip">
         <string>Optional note the recipient will see</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="placeholderText">
         <string>Description (optional)</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>7</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,0,0,3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="control::MwcLabelProgress" name="progress">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>PROG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>7</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabelProgress.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcComboBox</class>
   <extends>QComboBox</extends>
   <header>control_desktop/MwcComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelTiny</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcCheckBox</class>
   <extends>QCheckBox</extends>
   <header>control_desktop/MwcCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>accountComboBox</tabstop>
  <tabstop>amountEdit</tabstop>
  <tabstop>allAmountButton</tabstop>
  <tabstop>generatePoof</tabstop>
  <tabstop>sendEdit</tabstop>
  <tabstop>contactsButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
